src/runtimes/node/utils/entry_file.ts

```diff
@@ -12,28 +14,72 @@ const getEntryFileContents = (mainPath: string, moduleFormat: string) => {
   const importPath = `.${mainPath.startsWith('/') ? mainPath : `/${mainPath}`}`
 
   if (moduleFormat === ModuleFormat.COMMONJS) {
-    return `module.exports = require('${importPath}')`
+    return `module.exports=require('${importPath}')`
```

Comment:
Why are we changing this? Note that this will start creating functions with a different SHA for the same input, which means creating new Lambdas unnecessarily.

Reply:
Reverted, I though might be nice to save some bytes and because it is generated there is no need for codestyle, but I haven't thought about the hash implications. Good catch